test/perf/chacha20poly1305.test.ts

```diff
@@ -1,57 +1,60 @@
 import {itBench, setBenchOpts} from "@dapplion/benchmark";
 import crypto from "crypto";
 import {ChaCha20Poly1305 as ChaCha20Poly1305Stablelib} from "@stablelib/chacha20poly1305";
+import {chacha20poly1305 as noble} from '@noble/ciphers/chacha';
 import {ChaCha20Poly1305} from "../../src/chacha20poly1305";
-import {KEY_LENGTH, NONCE_LENGTH, TAG_LENGTH} from "../../common/const";
+import {KEY_LENGTH, NONCE_LENGTH} from "../../common/const";
 import {newInstance} from "../../src/wasm";
+import {formatBytes} from "./util";
 
 describe("chacha20poly1305", function () {
   this.timeout(0);
   setBenchOpts({
     minMs: 30_000,
   });
   const ctx = newInstance();
-  const asImpl = new ChaCha20Poly1305(ctx);
+  const chainsafe = new ChaCha20Poly1305(ctx);
 
   const key = new Uint8Array(crypto.randomBytes(KEY_LENGTH));
-  const jsImpl = new ChaCha20Poly1305Stablelib(key);
+  const stablelib = new ChaCha20Poly1305Stablelib(key);
   const nonce = new Uint8Array(crypto.randomBytes(NONCE_LENGTH));
-  // const jsImpl = new ChaCha20Poly1305Stablelib(key);
   const ad = new Uint8Array(crypto.randomBytes(32));
+  const runsFactor = 1000;
 
-  for (const dataLength of [512, 4096, 16384, 65_536, 524_288]) {
-    const plainText = new Uint8Array(crypto.randomBytes(dataLength));
-    const sealed = jsImpl.seal(nonce, plainText, ad);
-
-    itBench({
-      id: `as seal with data length ${dataLength}`,
-      fn: () => {
-        asImpl.seal(key, nonce, plainText, ad);
-      },
-    });
 
-    itBench({
-      id: `js seal with data length ${dataLength}`,
-      fn: () => {
-        jsImpl.seal(nonce, plainText, ad);
-      },
-    });
+  for (const dataLength of [32, 64, 1024, 4096, 8192, 65536]) {
+    const plainText = new Uint8Array(crypto.randomBytes(dataLength));
+    const sealed = stablelib.seal(nonce, plainText, ad);
+    const testCases: {impl: string; sealFn: () => void; openFn: () => void}[] = [
+      {impl: "chainsafe", sealFn: () => chainsafe.seal(key, nonce, plainText, ad), openFn: () => chainsafe.open(key, nonce, sealed, ad)},
+      {impl: "stablelib", sealFn: () => stablelib.seal(nonce, plainText, ad), openFn:  () => stablelib.open(nonce, sealed, ad)},
+      {impl: "noble", sealFn: () => noble(key, nonce, ad).encrypt(plainText), openFn: () => noble(key, nonce, ad).decrypt(sealed)}
+    ]
 
-    itBench({
-      id: `as open with data length ${dataLength}`,
-      beforeEach: () => new Uint8Array(sealed),
-      fn: (clonedSealed) => {
-        // overwriteSealed as true to avoid memory allocation
-        asImpl.open(key, nonce, clonedSealed, ad, clonedSealed.subarray(0, clonedSealed.length - TAG_LENGTH));
-      },
-    });
+    // seal
+    for (const {impl, sealFn} of testCases) {
+      itBench({
+        id: `${impl} seal ${formatBytes(dataLength)}`,
```

Comment:
Format in reverse to it's easier to read,
```suggestion
        id: `${formatBytes(dataLength)} seal ${impl}`,
```